telemetry/DevHome.Telemetry/DevHome.Telemetry.csproj

```diff
@@ -1,10 +1 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>net6.0-windows10.0.22000.0</TargetFramework>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
-    <RootNamespace>DevHome.Telemetry</RootNamespace>
-    <Platforms>x86;x64;arm64</Platforms>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
-    <UseWinUI>true</UseWinUI>
-  </PropertyGroup>
-</Project>
+<Project Sdk="Microsoft.NET.Sdk"><PropertyGroup><TargetFramework>net6.0-windows10.0.22000.0</TargetFramework><TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion><RootNamespace>DevHome.Telemetry</RootNamespace><Platforms>x86;x64;arm64</Platforms><RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers><UseWinUI>true</UseWinUI></PropertyGroup><ItemGroup><PackageReference Include="Microsoft.Telemetry.Inbox.Managed" Version="10.0.25148.1001-220626-1600.rs-fun-deploy-dev5" /></ItemGroup></Project>
```

Comment:
Can we put this formatting back? If this was a control KD thing I don't think we should do that on generated files.

Reply:
...That shouldn't have happened and I have no idea why it did.  I'll get it changed.